### ISSUE TITLE
dayMinWidth + Optimizing for Print - triggers when enabled/disabled

### ISSUE THREAD

**Issue by michalfordonski** [fullcalendar/fullcalendar#7097]
### Checklist

- [X] I've already searched through [existing tickets](https://github.com/fullcalendar/fullcalendar/issues)
- [X] Other people will find this feature useful

### Is this feature for a specific connector (React/Angular/etc)?

Other

### Feature Description

How can I catch the launch of print optimization?

When I use dayMinWidth the optimization cuts off the remaining resources on the printout. If I had the ability to get to the event before/after starting the printout and after closing it, I could manually control this parameter.

Something like resourceLabelDidMount is also for prints? If so, please let me know as I can't find it in the documentation. If not, is it possible for version 6.x to add it?

**Comment by acerix** [comment]
Do you mean you are wanting to set a different `dayMinWidth` for the print version?

There is the `beforeprint` callback before starting the printout:

https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeprint_event

**Comment by michalfordonski** [comment]
Yes, in my case it is enough - the width will be set correctly. However, if someone would like to act before / after printing, they may have a problem with the not fully controlled sequence of events. Then, without the appropriate trigger, which will be called when starting the printout via the calendar, there may be a problem. To sum up: Your proposal really helped me and can be closed. Thank you
